proxy/proxy.go

```diff
@@ -967,6 +967,14 @@ func (p *Proxy) serveResponse(ctx *context) {
 
 	start := time.Now()
 	copyHeader(ctx.responseWriter.Header(), ctx.response.Header)
+
+	if err := ctx.Request().Context().Err(); err != nil {
+		// deadline exceeded or canceled in stdlib, client closed request
+		// see https://github.com/zalando/skipper/pull/864
+		p.log.Infof("Client request: %v", err)
```

Comment:
Not sure if we want to log this. If the client closes the request, then the reason is not rooted in skipper. Isn't the the metrics on the status code 499 enough?